trinity/main.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import asyncio
+import atexit
+
+from evm.db.chain import BaseChainDB
+from evm.exceptions import CanonicalHeadNotFound
+from evm.chains.ropsten import (
+    ROPSTEN_GENESIS_HEADER,
+    ROPSTEN_NETWORK_ID,
+)
+from evm.chains.mainnet import MAINNET_VM_CONFIGURATION
+from evm.p2p import ecies
+from evm.p2p.lightchain import LightChain
+from evm.db.backends.level import LevelDB
+
+from trinity.utils.filesystem import (
+    ensure_path_exists,
+)
+from trinity.utils.logging import (
+    setup_trinity_logging,
+    setup_queue_logging,
+)
+from trinity.utils.mp import (
+    ctx,
+)
+from trinity.utils.xdg import (
+    get_data_dir,
+)
+
+
+DEFAULT_LOG_LEVEL = 'info'
+LOG_LEVEL_CHOICES = (
+    'debug',
+    'info',
+)
+
+
+parser = argparse.ArgumentParser(description='Trinity')
+parser.add_argument(
+    '-l',
+    '--log-level',
+    choices=LOG_LEVEL_CHOICES,
+    default=DEFAULT_LOG_LEVEL,
+)
+
+
+DemoLightChain = LightChain.configure(
+    name='Demo LightChain',
+    privkey=ecies.generate_privkey(),  # TODO: get from stored key.
+    vm_configuration=MAINNET_VM_CONFIGURATION,
+    network_id=ROPSTEN_NETWORK_ID,
+)
+
+
+def main():
+    args = parser.parse_args()
+
+    logger, log_queue, listener = setup_trinity_logging(args.log_level.upper())
+
+    # start the listener thread to handle logs produced by other processes in
+    # the local logger.
+    listener.start()
+
+    db_path = get_data_dir('ropsten')
```

Comment:
We should make a constant for this

Reply:
If you mean the string `'ropsten'`, yes, 👍